libp2p/protocols/connectivity/relay/client.nim

```diff
@@ -170,35 +167,33 @@ proc dialPeerV2*(
     dstAddrs: seq[MultiAddress]): Future[Connection] {.async.} =
   let
     p = Peer(peerId: dstPeerId, addrs: dstAddrs)
-    pb = encode(HopMessage(msgType: HopMessageType.Connect, peer: some(p)))
+    pb = encode(HopMessage(msgType: HopMessageType.Connect, peer: Opt.some(p)))
 
   trace "Dial peer", p
 
   let msgRcvFromRelay = try:
     await conn.writeLp(pb.buffer)
-    HopMessage.decode(await conn.readLp(RelayClientMsgSize)).get()
+    HopMessage.decode(await conn.readLp(RelayClientMsgSize)).tryGet()
   except CancelledError as exc:
     raise exc
-  except CatchableError as exc:
+  except ValueError as exc:
```

Comment:
is this right?

Reply:
@lchenut ?

Reply:
Yes it is right, the only exception that can be raise here are the CancelledError (if await fails) and ValueError (if tryGet fails).

Reply:
readLp can fail with another error though?

Reply:
Hum... You're right. I'm dumb. Everything is fine.